robot/2023/src/main/java/frc/robot/Robot.java

```diff
@@ -76,6 +84,10 @@ public void robotInit() {
     parkingCamera = CameraServer.startAutomaticCapture(0);
     leftBackCamera = CameraServer.startAutomaticCapture(1);
     rightBackCamera = CameraServer.startAutomaticCapture(2);
+
+    //Encoder's distance/pulse
+    leftEncoder.setDistancePerPulse(1./256.);
```

Comment:
Obvs, we need to know what the actual number is :-/

Reply:
Oh, and this should be a final double at the top or something and not a magic number here.

Reply:
Set to constant "ENCODER_DISTANCE_PER_PULSE"

Reply:
Though we still need to calculate what the value will actually be